### ISSUE TITLE
Internal Error

### ISSUE THREAD

**Issue by ThatOneShortGuy** [microsoft/pylance-release#5040]
## Environment data
-   Language Server version: v2023.10.50
-   OS and version: Windows 10.0.19045
-   Python version (& distribution if applicable, e.g. Anaconda): Python 3.9.13 64-bit

## Code Snippet
The entire file contains:
```python
import os
```
or
```py
print("Hello World!")
```
or
```py
1 + 3
```
or even
```py

```

## Expected behavior
Does not error on usage. I need to just turn off auto-update, and this won't be an issue anymore...

## Actual behavior
This error is a constant error that happens on any valid or non valid python file.
It unfortunately updated and broke everything.

My workaround is to use version v2023.10.40

## Logs
```
An internal error occurred while parsing file "E:\Data\dataloader.py": TypeError: _0x2f33cc[(_0x1efd68(...) + _0x1efd68(...))] is not a function
    at _0xa6e677.dirExistsCached (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\server.bundle.js:1:902382)
    at _0xa6e677._resolveAbsoluteImport (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:303694)
    at _0xa6e677.resolveAbsoluteImport (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:296859)
    at _0xa6e677._resolveBestAbsoluteImport (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:305905)
    at _0xa6e677._resolveImportStrict (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:299332)
    at _0xa6e677.resolveImportInternal (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:294521)
    at _0xa6e677.resolveImport (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:290200)
    at _0xa6e677.resolveImport (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\server.bundle.js:1:898287)
    at i (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:510945)
    at _0x4c3a07._resolveImports (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:511197)
    at c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:499381
    at a.timeOperation (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:1129990)
    at c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:499348
    at t.LogTracker.log (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:1092230)
    at _0x4c3a07.parse (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:498098)
    at N._parseFile (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:434006)
    at N._bindFile (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:434964)
    at N.getBoundSourceFileInfo (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:420286)
    at N.getBoundSourceFile (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:420039)
    at _0x2a1885.<computed> (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\server.bundle.js:1:786498)
    at c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\server.bundle.js:1:137241
    at t.LogTracker.log (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:1092230)
    at c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\server.bundle.js:1:137107
    at _0x5f4a97.<computed> (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\server.bundle.js:1:367551)
    at c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\server.bundle.js:1:136489
    at t.run (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:1028131)
    at _0x79f29.onMessage (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\server.bundle.js:1:136200)
    at _0x79f29._onMessageWrapper (c:\Users\braxt\.vscode\extensions\ms-python.vscode-pylance-2023.10.50\dist\pyright.bundle.js:1:1023516)
    at MessagePort.[nodejs.internal.kHybridDispatch] (node:internal/event_target:735:20)
    at MessagePort.exports.emitMessage (node:internal/per_context/messageport:23:28)
```

**Comment by debonte** [comment]
Duplicate of https://github.com/microsoft/pylance-release/issues/5009

**Comment by xianmsb** [comment]
i find it too, thanks

**Comment by debonte** [comment]
@xianmsb, this is a duplicate of #5009 which is fixed in our latest prerelease build (2023.10.53).
